seurat-filter-cells.R

```diff
@@ -95,7 +95,21 @@ if (! is.null(cells_use)){
 
 suppressPackageStartupMessages(require(Seurat))
 
-filtered_seurat_object <- FilterCells(seurat_object, subset.names = subset_names, low.thresholds = lt, high.thresholds = ht, cells.use = cells_use)
```

Comment:
So there's no FilterCells function any more?

Reply:
Nope... I wonder if we should transform this in a single module for filtering, as the new setup allows to filter genes on the same subset function.

Reply:
That being the case, I wonder if we should use the way this is done in scater-scripts: https://github.com/ebi-gene-expression-group/bioconductor-scater-scripts/blob/develop/scater-filter.R. That doesn't use a specific function, so we could maybe adapt it for Seurat objects>

Reply:
(but I don't want to create lots of extra work)